src/ext/java/org/opentripplanner/ext/siri/SiriTimetableSnapshotSource.java

```diff
@@ -302,7 +308,7 @@ public void applyEstimatedTimetable(
                 }
               } else {
                 // Updated trip
-                if (handleModifiedTrip(transitModel, fuzzyTripMatcher, feedId, journey)) {
+                if (handleModifiedTrip(transitModel, fuzzyTripMatcher, feedId, journey).isEmpty()) {
```

Comment:
This still has this check

Reply:
This method returns a `List<UpdateError>`. Would you prefer it to return a `Result<Void, List<UpdateError>>`?

After having thought about it for a bit I think that using a `Result` is correct, even though it's a slightly overcomplicated type. WDYT?